src/components/Accordion/AccordionItem/AccordionItem.scss

```diff
@@ -21,6 +22,10 @@ $accordionHeader_height: 48px;
     &.moonstone-reversed {
         border-color: var(--color-gray);
     }
+
+    &:has(.moonstone-accordionItem_header:focus-visible) {
```

Comment:
Should it be better to apply the style directly to `moonstone-accordionItem_header` instead of the parent?
In that case we will have to override `outline-offset` to `-2px`